_posts/2019-08-06-pre-post-events-sustainable-trains.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title: "Pre-/Post-Events: sustainable train travels from and to RustFest Barcelona"
+authors:
+  - dns2utf8
+socialImageSrc: "/assets/posts/barcelona/cfp-opens.png"
```

Comment:
Needs updated social image.

Reply:
Yes, I was thinking the map

Reply:
![grafik](https://user-images.githubusercontent.com/739070/62707014-28cf5500-b9e0-11e9-92dc-39ee632a2ec6.png)